makefiles/package.mk

```diff
@@ -56,7 +56,7 @@ ifeq ($(_THEOS_FINAL_PACKAGE),$(_THEOS_TRUE))
 VERSION.EXTRAVERSION = $(if $(PACKAGE_BUILDNAME),+$(PACKAGE_BUILDNAME))
 _THEOS_PACKAGE_DEFAULT_VERSION_FORMAT = $(THEOS_PACKAGE_BASE_VERSION)$(VERSION.EXTRAVERSION)
 else
-VERSION.INC_BUILD_NUMBER = $(shell THEOS_PROJECT_DIR="$(THEOS_PROJECT_DIR)" "$(THEOS_BIN_PATH)/package_version.sh" -N "$(THEOS_PACKAGE_NAME)" -V "$(__BASEVER_FOR_BUILDNUM)")
+VERSION.INC_BUILD_NUMBER = $(shell bash THEOS_PROJECT_DIR="$(THEOS_PROJECT_DIR)" "$(THEOS_BIN_PATH)/package_version.sh" -N "$(THEOS_PACKAGE_NAME)" -V "$(__BASEVER_FOR_BUILDNUM)")
 _THEOS_PACKAGE_DEFAULT_VERSION_FORMAT = $(THEOS_PACKAGE_BASE_VERSION)-$(VERSION.INC_BUILD_NUMBER)$(VERSION.EXTRAVERSION)
```

Comment:
I'm somewhat surprised that we ended up shipping "`FINALPACKAGE`". The whole point of the composable versioning system was to make that second part not required ;P

Reply:
it also doesn't seem safe to assume `bash` everywhere. Especially if the script isn't using any bashisms?

Reply:
The shebang in the script is #!/usr/bin/bash which is why I used bash

Reply:
I've been hoping to revisit the FINALPACKAGE implementation, it hasn't changed since I merged rpetrich's branch a few years ago. I figured it was convenient shorthand for "I'm releasing this", and it also calls `plutil -convert binary1` and `pincrush` (that too needs revisiting…). 

We force SHELL = bash at the top of common.mk, I don't know of any realistic Unix-alike system that doesn't ship bash, and I haven't seen anyone try to run Theos on anything more interesting than macOS/iOS or Linux/Linux-alike. So it's safe to assume bash as we already have a hard dependency on it anyway.

Reply:
It should probably be $(SHELL) instead of "bash" then - I missed that variable.